deployment/deploy.js

```diff
@@ -1,14 +1,44 @@
 const etherlime = require('etherlime-lib');
-const LimeFactory = require('../build/LimeFactory.json');
-
+const MyContract = require('../build/MyContract.json');
+const LinkToken = require('../build/LinkToken.json');
+const Oracle = require('../build/Oracle.json');
+const config = require('../config.json');
 
 const deploy = async (network, secret, etherscanApiKey) => {
 
-	const deployer = new etherlime.EtherlimeGanacheDeployer();
-	const result = await deployer.deploy(LimeFactory);
+	let deployer;
+	let contract;
+	let linkToken;
+	let oracle;
+
+	if (!config.deployAndVerify) {
```

Comment:
I'd rather to make this via two deployment files - deploy.js for ganache and deploy-ropsten.js for ropsten (that includes verification)

one can use them by specifing it in the depoy command `etherlime deploy --file deployment/deploy-ropsten.js`

Reply:
Also we can add this to the npm scripts too :)